### ISSUE TITLE
When I run RXPTest from the latest master I get an unresolved module

### ISSUE THREAD

**Issue by gmangov** [Microsoft/reactxp#817]
Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `F:\src\react_xp_my_fork\samples\RXPTest\node_modules\react-native\Libraries\re
act-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

**Comment by erictraut** [comment]
I'm not able to repro the problem. Please:
1. Make sure you have a recent version of node installed
2. Delete the node_modules directory under RXPTest
3. Run "npm i" and "npm run rn-watch" to build

**Comment by gmangov** [comment]
It reproes consistently for me. However when I matched "react" and "react-native" versions in RXPTest packages with the corresponding from react-native-windows package, the issue got resolved.

**Comment by reseul** [comment]
RXPTest/UWP is not in a stable state due to the various movement speeds of dependencies and the delay of the RNW release due to the haste changes (you're seeing a symptom of that).
A 0.57-rc.0 RNW was released yesterday, so I'll be able to align/test RXPTest/UWP as soon as the RN dependency moves to a 0.57 version.
